### PR TITLE
fix(json-schema-check): update schemas

### PR DESCRIPTION
**Pull request by xmfcx**
## Description

Part of:
- https://github.com/autowarefoundation/autoware.universe/issues/6473

## Tests performed

Not applicable.

## Effects on system behavior

Not applicable.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by xmfcx** [comment]
CI is green :slightly_smiling_face: https://github.com/autowarefoundation/autoware.universe/actions/runs/7993331975/job/21828904972#step:3:43

But I see that there are some contested problems. I will separate them out of this PR, only leave the simple changes here.

**Comment by xmfcx** [comment]
I have only updated the schema files on this PR, makes the CI greener than before:

**Before:**
- https://github.com/autowarefoundation/autoware.universe/actions/runs/7993422751/job/21829219975#step:3:43

**After:**
- https://github.com/autowarefoundation/autoware.universe/actions/runs/7993389878/job/21829103578?pr=6474#step:3:43

Very easy to review, there shouldn't be any risk.

File: planning/behavior_velocity_planner/schema/behavior_velocity_planner.schema.json [hunk 1]
```diff
@@ -50,6 +55,7 @@
       "required": [
         "forward_path_length",
         "backward_path_length",
+        "behavior_output_path_interval",
```
Comment (xmfcx) [inline planning/behavior_velocity_planner/schema/behavior_velocity_planner.schema.json:58]: @takayuki5168 I've added the behavior_output_path_interval update after your PR:
- https://github.com/autowarefoundation/autoware.universe/commit/fdeb24c3650901ab7f25223fb0c131afd824bff4

Please review this part :bowing_man:

**Comment by xmfcx** [comment]
And @badai-nguyen could you review the compare_map_segmentation schema updates?

Since you have also reviewed:
- https://github.com/autowarefoundation/autoware.universe/pull/5005

File: planning/behavior_velocity_planner/schema/behavior_velocity_planner.schema.json (hunk 1 above)
Comment (takayuki5168) [reply planning/behavior_velocity_planner/schema/behavior_velocity_planner.schema.json:58]: Sorry, I've just merged my PR which also includes the vehicle side.
https://github.com/autowarefoundation/autoware.universe/pull/6456

Please rebase it. Thank you in advance.

Comment (xmfcx) [reply planning/behavior_velocity_planner/schema/behavior_velocity_planner.schema.json:59]: @takayuki5168 thanks.
I've responded to the one related to `raw_vehicle_cmd_converter.schema.json` in:
- https://github.com/autowarefoundation/autoware.universe/pull/6456#discussion_r1499116775

Rebased, and updated the description for the one you've provided, please review again!

Comment (takayuki5168) [reply planning/behavior_velocity_planner/schema/behavior_velocity_planner.schema.json:59]: Thank you so much!

**Review by takayuki5168 — APPROVED** [review]

**Comment by xmfcx** [comment]
@badai-nguyen could you review the compare_map_segmentation part please?

File: planning/behavior_velocity_planner/schema/behavior_velocity_planner.schema.json [hunk 2]
```diff
@@ -56,6 +56,7 @@
         "forward_path_length",
         "behavior_output_path_interval",
         "backward_path_length",
+        "behavior_output_path_interval",
```
Comment (xmfcx) [inline planning/behavior_velocity_planner/schema/behavior_velocity_planner.schema.json:59]:
```suggestion
```

**Comment by xmfcx** [comment]
@badai-nguyen could you review the compare_map_segmentation part please?

**Review by badai-nguyen — APPROVED** [review]

**Comment by xmfcx** [comment]
Thank you so much!

**Pull request merged**